.editorconfig

```diff
@@ -35,6 +35,9 @@ insert_final_newline = true
 
 # Dotnet code style settings:
 [*.cs]
+dotnet_style_allow_statement_immediately_after_block_experimental = false:warning
+dotnet_style_allow_multiple_blank_lines_experimental = false:warning
```

Comment:
There two are defined below, so will have no effect

Reply:
Good catch. The existing entries didn't seem to actually be doing anything which I guess is why I missed them.